dev/releases/release_notes.py

```diff
@@ -104,15 +105,12 @@ def warning(s):
 
 def get_tag_date(tag: str) -> str:
     if is_existing_tag(tag):
-        res = subprocess.run(
-            ["git", "for-each-ref", "--format=%(creatordate:short)", "refs/tags/" + tag],
-            check=True,
-            capture_output=True,
-            text=True,
-        )
+        res = subprocess.run(f"""gh release view {tag} --json=createdAt""", shell=True,
```

Comment:
Isn't this brittle, e.g. if `tag` contains whitespace it breaks? The julia syntax for shell code is pretty cool in that ``run(`gh release view $tag`)`` is handled in a clever way, being tokenized *before* substitutions. But I don't think Python does that. Letting a shell parse the string feels like a step backwards